docs/user-guide/mvd-editor.md

```diff
@@ -2,40 +2,71 @@
 
 With the Zowe Editor, you can create and edit the many types of files.
 
-## Specifying a language server
+## Specifying a highlighting language
 
-To specify a language server, complete these steps:
+1. Click **Language** on the editor toolbar. A dropdown menu will be displayed.
+2. From the dropdown, select the desired language. Plain Text will be chosen by default.  
```

Comment:
"Plain Text will be chosen by default.  "

should be

"Plain Text will be chosen by default if automatic language detection is not able to determine the language. "  
We have simple checkers in place to check for z/OS languages like JCL and ASM 😃